src/views/pages/accounts/LoginPageTs.ts

```diff
@@ -204,6 +208,24 @@ export default class LoginPageTs extends Vue {
    * Process login request.
    * @return {void}
    */
+
+  isNotHardwareWallet(wallet : any):boolean{
+    let flag2 =false
+    
+    const existingWallets = Object.values(this.walletsRepository.collect())
+    existingWallets.filter(w=>{
+      if (w.getIdentifier()==wallet){
+        if (w.values.get("type") !== WalletType.fromDescriptor('Ledger')  ) {
+        
+        flag2 = true
+      } 
+      else flag2 = false
+    }
+  }
+  )
+    return flag2
+  }
```

Comment:
- use non-negative word (`isHardwareWallet`)
- use `existingWallets.find()` to move iterator instead of iterating manually
- return flag directly, no need for variable